api/docs/open_api_schema.json

```diff
@@ -1358,6 +1358,18 @@
             ],
```

Comment:
file automatically modified by `./run.py api update-schemas`

Reply:
Remember to re-run this if you make my above tweaks. 😁